gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/entities/Task.java

```diff
@@ -90,10 +107,11 @@ public ProcessFile getProcessFile(String fileType) {
                 .orElseThrow(() -> new RuntimeException(String.format("Queried fileType does not exist %s", fileType)));
     }
 
-    public static TaskDto createDtofromEntity(Task task) {
+    public static TaskDto createDtoFromEntity(Task task) {
         return new TaskDto(
-            task.getTimestamp(),
-            task.getStatus(),
-            task.getProcessFiles().stream().map(ProcessFile::createDtofromEntity).collect(Collectors.toList()));
+                task.getTimestamp(),
+                task.getStatus(),
+                task.getProcessFiles().stream().map(ProcessFile::createDtofromEntity).collect(Collectors.toList()),
+                task.getProcessEvents().stream().map(ProcessEvent::createDtoFromEntity).collect(Collectors.toList()));
```

Comment:
Same, maybe change your editor settings for tabulations

Reply:
same

Reply:
You can still check for indentation ! Otherwise we'll always get diff just for formatting issues